spec/controllers/replies_controller_spec.rb

```diff
@@ -61,13 +61,33 @@
         expect(controller.gon.editor_user[:username]).to eq(user.username)
         # templates
         templates = assigns(:templates)
-        expect(templates.length).to eq(2)
-        template = templates.first
-        expect(template).to eq(char2.template)
+        expect(templates.length).to eq(3)
+        used = templates.first
+        expect(used.name).to eq("Thread characters")
```

Comment:
Hrrrm, do we want this? It's probably a pain to work around, and it's probably okay as is.